### PR TITLE
Feature/uat fixes

### PR DESCRIPTION
**Pull request by ntoombs19**

### CODE REVIEW

File: web/css/source/rebar/_fluid_typography.less [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@import 'lib/variables/_fluid_typography.less';
```
Comment (erikhansen) [inline web/css/source/rebar/_fluid_typography.less]: Why did you pull in the fluid typography code? Did you determine it was necessary in order to address the typography on the site?

File: MagentoEse_LookBook/web/css/source/_extend.less [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
```
Comment (erikhansen) [inline MagentoEse_LookBook/web/css/source/_extend.less]: Would this CSS not belong better in the Lookbook module itself?

File: web/css/source/rebar/_fluid_typography.less (hunk 1 above)
Comment (ntoombs19) [inline web/css/source/rebar/_fluid_typography.less]: Not absolutely necessary but it styles the header to an appropriate size while making them a fluid size. The alternative is to style the header sizes individually at each breakpoint.

File: MagentoEse_LookBook/web/css/source/_extend.less (hunk 2 above)
Comment (ntoombs19) [inline MagentoEse_LookBook/web/css/source/_extend.less]: It used to be but that was the source of the compilation error. The variable @font-family-name__secondary is defined in the theme not in the module so the module didn't have access to the variable.

Comment (erikhansen) [inline MagentoEse_LookBook/web/css/source/_extend.less]: Ah, ok.

File: web/css/source/rebar/_fluid_typography.less (hunk 1 above)
Comment (erikhansen) [inline web/css/source/rebar/_fluid_typography.less]: Got it.

**Pull request merged**